src/@chakra-ui/gatsby-plugin/components/components.utils.ts

```diff
@@ -33,7 +32,6 @@ export {
   avatarDefaultTheme,
   badgeDefaultTheme,
   breadcrumbDefaultTheme,
-  buttonDefaultTheme,
```

Comment:
@pettinarip The `FeedbackWidget` is using a button to [close the modal](https://github.com/ethereum/ethereum-org-website/blob/56011e47b16be17cf9967288581f6a32a3eb6113/src/components/FeedbackWidget.tsx#L220) and is applying the ghost variant.

This actually needs a PR to switch from the `Button` to `CloseButton` component for the widget and thereby remove the need to declare a ghost variant. (CloseButton is not consuming the `Button` theme)

Otherwise, I believe no other theme defaults are needed for `Button`.

Reply:
@pettinarip the default theming is not being consumed.

The `solid` variant is [chosen as the default](https://github.com/ethereum/ethereum-org-website/pull/10315/files#diff-b945fbe9833777dc2b592b0bcf21edbe688bf9e6efe990265f0e6a59cd049da7R125), so the `ghost` variant needs to be declared as an empty object so that only the base styles are being rendered.

Reply:
Aha! Got it now, thanks.